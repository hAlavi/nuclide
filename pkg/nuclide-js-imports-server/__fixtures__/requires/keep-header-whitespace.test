/**
 * @formout
 */

'use strict';

jest
  .enableAutomock()

const x = Y;
