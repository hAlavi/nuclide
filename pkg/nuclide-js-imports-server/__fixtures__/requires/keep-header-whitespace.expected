/**
 * @formout
 */

'use strict';

jest
  .enableAutomock()

const Y = require('Y');

const x = Y;
